requirements.txt

```diff
@@ -39,9 +40,9 @@ pytz==2022.6.*
 requests==2.28.*
 scikit-learn==1.2.*
 scikit-image==0.22.*
-scipy==1.9.*
```

Comment:
If you're changing versions...

Here's what I had to change to be able to use scilpy on Beluga. Can we put scipy 1.11?

![image](https://github.com/scilus/scilpy/assets/4967417/92518108-2528-4889-864b-8cd01b6015c9)